### ISSUE TITLE
MacOS `bash: ./from-template.sh: No such file or directory`

### ISSUE THREAD

**Issue by spdegabrielle** [nixin72/from-template#3]
I'm using this issue to log my testing 

```bash
% raco from-template guiapp
bash: ./from-template.sh: No such file or directory
#f
```

(About to go out to the SS Great Britain in Bristol so I won't get back to this till the evening )

**Comment by spdegabrielle** [comment]
Update

```racket
(define-runtime-path macosx-script "from-template.sh")

(define [clone-repo repo-name dir-name]
  (case (system-type 'os)
    [(unix)
     (system (string-append "bash ./from-template.sh " repo-name " " dir-name))]
    [(macosx)
     (system (string-append "bash " (path->string macosx-script) " " repo-name " " dir-name))]
    [(windows)
     (system (string-append (path->string windows-script) " " repo-name " " dir-name))])) 
```

returns

```
spdegabrielle@Miriams-MacBook-Pro testing-from-template % raco from-template guiapp
guiapp
/Users/spdegabrielle/Dev/testing-from-template/
Cloning into 'guiapp'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
/Users/spdegabrielle/Dev/from-template/from-template.sh: line 9: cd: guiapp: No such file or directory
#t
spdegabrielle@Miriams-MacBook-Pro testing-from-template % 

```

**Comment by spdegabrielle** [comment]
Resolved by #4
